helm/kanister-operator/templates/deployment.yaml

```diff
@@ -14,6 +14,12 @@ spec:
       labels:
 {{ include "kanister-operator.helmLabels" . | indent 8}}
     spec:
+{{- if .Values.podSecurityContext }}
+      securityContext: 
+        runAsUser: {{ .Values.podSecurityContext.runAsUser }}
+        fsGroup: {{ .Values.podSecurityContext.fsGroup }}
+        runAsNonRoot: {{ .Values.podSecurityContext.runAsNonRoot }}
```

Comment:
is there a reason because of which we would just want to support these 3 fields under `securityContext` or should we support configuring all the fields under securityContext?